docs-sphinx/_toc.yml

```diff
@@ -49,14 +47,10 @@ subtrees:
                         title: "Arrays of missing data"
                       - file: user-guide/how-to-create-strings
                         title: "Arrays of strings"
-                      - file: user-guide/how-to-create-lazy
-                        title: "Lazy arrays [todo]"
                       - file: user-guide/how-to-create-unflatten-group
                         title: "Unflattening and grouping [todo]"
                       - file: user-guide/how-to-create-arraybuilder
                         title: "ArrayBuilder (easy & general)"
-                      - file: user-guide/how-to-create-layoutbuilder
```

Comment:
This is also gone!

Reply:
@ManasviGoyal's documentation will appear somewhere, maybe in a different part of the documentation because it's for a different set of users (C++).

With the LayoutBuilder section gone, the text "(easy & general)" is no longer serving a function: it used to be the thing that distinguished ArrayBuilder from LayoutBuilder: ease/generality versus performance. Now the distinction is even larger: Python versus C++. (Note that @ianna will be working on a LayoutBuilder for Numba, so in general I should say "compiled.")

Reply:
Hmm, I still think this applies Jim; array-builder is intuitive (just build a tree using a builder pattern) vs layouts, which are "advanced". Are we OK to leave this, and revisit it in the next wave of docs updates?